python-interface/src/MiniBotFramework/Sensing/Sensor.py

```diff
@@ -2,6 +2,7 @@
 class Sensor:
     def __init__(self, bot, name):
         self.name = name
+        # print "Sensor init called! Name: " + name
```

Comment:
this might be a better idea than printing in MiniBot.py - it helps minimize the size of MiniBot.py.